src/+soilmoisture/calculateBoundaryConditions.m

```diff
@@ -12,21 +12,41 @@
 
     Precip = InitialValues.Precip;
     Precip_msr = ForcingData.Precip_msr;
-
     Precipp = 0;
-    %  Apply the bottom boundary condition called for by BoundaryCondition.NBChB
-    if BoundaryCondition.NBChB == 1            %  Specify matric head at bottom to be ---BoundaryCondition.BChB;
-        RHS(1) = BoundaryCondition.BChB;
-        C4(1, 1) = 1;
-        RHS(2) = RHS(2) - C4(1, 2) * RHS(1);
-        C4(1, 2) = 0;
-        C4_a(1) = 0;
-    elseif BoundaryCondition.NBChB == 2        %  Specify flux at bottom to be ---BoundaryCondition.BChB (Positive upwards);
-        RHS(1) = RHS(1) + BoundaryCondition.BChB;
-    elseif BoundaryCondition.NBChB == 3        %  BoundaryCondition.NBChB=3,Gravity drainage at bottom--specify flux= hydraulic conductivity;
-        RHS(1) = RHS(1) - SoilVariables.KL_h(1, 1);
+    
+    %  Apply the bottom boundary condition called for by BoundaryCondition.NBChB	
+    GroundwaterSettings = io.readGroundwaterSettings() % added by Mostafa
```

Comment:
This line should be added to the main script `STEMMUS_SCOPE.m` and only the variable `GroundwaterSettings` should be added as an input argument to the function `calculateBoundaryConditions`.